charts/jaeger-operator/templates/deployment.yaml

```diff
@@ -36,6 +36,11 @@ spec:
         - name: {{ include "jaeger-operator.fullname" . }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
+          securityContext:
```

Comment:
If you don't mind a small comment: I personally prefer the flexibility of setting the entire securityContext via values.yaml, e.g.:

```yaml
# values.yaml:
securityContext:
  allowPrivilegeEscalation: false
  capabilities:
    drop:
      - ALL
  readOnlyRootFilesystem: true
  runAsNonRoot: true
  privileged: false
  runAsUser: 65532
```
with deployment.yaml being:

```yaml
          securityContext:
            {{- toYaml .securityContext | nindent 12 }}
```

Reply:
Sure. However, since there is already `securityContext` at pod level, it would be better to add `containerSecurityContext` in `values.yml`